packages/liveblocks-node/src/client.ts

```diff
@@ -929,6 +930,38 @@ export class Liveblocks {
     }
     return (await res.json()) as Promise<CommentData>;
   }
+
+  /**
+   * Creates a comment.
+   * 
+   * @param params.roomId The room ID to create the comment in.
+   * @param params.threadId The thread ID to create the comment in.
+   * @returns The created comment.
+   */
+  public async createComment(params: {
+    roomId: string;
+    threadId: string;
+    comment: {
+      userId: string;
+      /**
+       * ISO 8601 date string
+       */
+      createdAt?: string;
```

Comment:
This should be a `Date`. We decided to transform all the ISO 8601 strings to `Date` when fetching so we should do the same when creating an object.